tests/test_ext.py

```diff
@@ -162,10 +162,13 @@ def interpolate(self, token):
 
 class TestExtensions:
     def test_extend_late(self):
-        env = Environment()
-        env.add_extension("jinja2.ext.autoescape")
-        t = env.from_string('{% autoescape true %}{{ "<test>" }}{% endautoescape %}')
-        assert t.render() == "&lt;test&gt;"
+        with pytest.deprecated_call():
```

Comment:
None of these tests are actually using deprecated functionality, it's just the unneeded `autoescape` extension that's causing the warning. Remove the use of `'jinja2.ext.autoescape'` instead of wrapping with `deprecated_call()`.